frontend/sozisel-app/src/graphql/queries/session.graphql

```diff
@@ -6,4 +6,56 @@ query SearchSessions($input: SearchSessionsInput!){
         scheduledStartTime
         startTime
     }
+}
+
+query SessionDetails($id: ID!){
+    session(id: $id){
+        id
+        name
+        scheduledStartTime
+        entryPassword
+        useJitsi
+        sessionTemplate {
+            id
+            estimatedTime
+            isPublic
+            name
+            owner{
+                id
+                firstName
+                lastName
+                email
+            }
+            agendaEntries{
+                id
+                name
+                startMinute
+            }
+            events {
+                id
+                name
+                startMinute
+                eventData {
+                    ... on Quiz {
```

Comment:
to make this query more readable you could use graphql fragments (a separate fragment for a quiz).